<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1440</width>
    <height>900</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CALFEM VTK Visualization Tool</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>1441</width>
      <height>851</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <item row="0" column="0">
      <widget class="QTabWidget" name="tabWidget">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>1</horstretch>
         <verstretch>1</verstretch>
        </sizepolicy>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="tab_shape">
        <attribute name="title">
         <string>Rendering</string>
        </attribute>
        <widget class="QWidget" name="horizontalLayoutWidget">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>0</y>
           <width>1411</width>
           <height>821</height>
          </rect>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="1">
           <widget class="QFrame" name="frame">
            <property name="cursor">
             <cursorShape>OpenHandCursor</cursorShape>
            </property>
            <property name="frameShape">
             <enum>QFrame::NoFrame</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Plain</enum>
            </property>
            <property name="lineWidth">
             <number>0</number>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
       <widget class="QWidget" name="tab_element">
        <attribute name="title">
         <string>Plots</string>
        </attribute>
        <widget class="QWidget" name="horizontalLayoutWidget_2">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>0</y>
           <width>1411</width>
           <height>821</height>
          </rect>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="1">
           <widget class="QFrame" name="frame_2">
            <property name="cursor">
             <cursorShape>OpenHandCursor</cursorShape>
            </property>
            <property name="frameShape">
             <enum>QFrame::NoFrame</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Plain</enum>
            </property>
            <property name="lineWidth">
             <number>0</number>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1440</width>
     <height>22</height>
    </rect>
   </property>
   <property name="defaultUp">
    <bool>true</bool>
   </property>
   <widget class="QMenu" name="menuTest">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuImport">
     <property name="title">
      <string>Import...</string>
     </property>
     <addaction name="actionMatlab_data"/>
     <addaction name="actionVTK_data"/>
    </widget>
    <widget class="QMenu" name="menuExport_2">
     <property name="title">
      <string>Export...</string>
     </property>
     <addaction name="actionTo_Matlab"/>
     <addaction name="actionTo_VTK"/>
    </widget>
    <addaction name="menuImport"/>
    <addaction name="menuExport_2"/>
   </widget>
   <widget class="QMenu" name="menuTest2">
    <property name="title">
     <string>View</string>
    </property>
    <widget class="QMenu" name="menuGeometry">
     <property name="title">
      <string>Mesh...</string>
     </property>
     <addaction name="actionShow_Elements"/>
     <addaction name="actionShow_Elements_2"/>
     <addaction name="actionShow_Dofs"/>
    </widget>
    <widget class="QMenu" name="menuDeformed_Mesh">
     <property name="title">
      <string>Deformed Mesh...</string>
     </property>
     <addaction name="actionShow_Elements_3"/>
     <addaction name="actionShow_Nodes"/>
     <addaction name="actionShow_Dofs_2"/>
    </widget>
    <addaction name="actionReset_Camera"/>
    <addaction name="separator"/>
    <addaction name="actionWireframe"/>
    <addaction name="separator"/>
    <addaction name="actionShow_Axis"/>
    <addaction name="actionShow_Origin"/>
    <addaction name="actionShow_Grid"/>
    <addaction name="actionShow_Scale"/>
    <addaction name="actionShow_Boundary_Conditions"/>
    <addaction name="separator"/>
    <addaction name="menuGeometry"/>
    <addaction name="menuDeformed_Mesh"/>
   </widget>
   <widget class="QMenu" name="menuMode">
    <property name="title">
     <string>Mode</string>
    </property>
    <addaction name="actionDefault"/>
    <addaction name="actionTrackball"/>
    <addaction name="actionActor"/>
    <addaction name="actionSelection"/>
   </widget>
   <widget class="QMenu" name="menuExport">
    <property name="title">
     <string>Select</string>
    </property>
    <addaction name="actionExport_Matlab_data"/>
    <addaction name="actionExport_VTK_data"/>
    <addaction name="separator"/>
    <addaction name="actionShow_browser"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionCALFEM_for_Python_documentation"/>
    <addaction name="actionVTK_tool_Guide"/>
    <addaction name="actionKeyboard_controls"/>
   </widget>
   <widget class="QMenu" name="menuAnimate">
    <property name="title">
     <string>Animate</string>
    </property>
    <addaction name="actionStart_Animation"/>
    <addaction name="actionEdit_Parameters"/>
   </widget>
   <addaction name="menuTest"/>
   <addaction name="menuExport"/>
   <addaction name="menuTest2"/>
   <addaction name="menuMode"/>
   <addaction name="menuAnimate"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionReset_Camera">
   <property name="text">
    <string>Reset Camera</string>
   </property>
  </action>
  <action name="actionWireframe">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Toggle Wireframe</string>
   </property>
  </action>
  <action name="actionShow_Axis">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show Axis</string>
   </property>
  </action>
  <action name="actionShow_Grid">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show Grid</string>
   </property>
  </action>
  <action name="actionActor">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Actor</string>
   </property>
  </action>
  <action name="actionDefault">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Default</string>
   </property>
  </action>
  <action name="actionExport_Matlab_data">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Element by ID...</string>
   </property>
  </action>
  <action name="actionExport_VTK_data">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Node by ID...</string>
   </property>
  </action>
  <action name="actionShow_Scale">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Show Scale</string>
   </property>
  </action>
  <action name="actionShow_Boundary_Conditions">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Show Boundary Conditions</string>
   </property>
  </action>
  <action name="actionMatlab_data">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Matlab data...</string>
   </property>
  </action>
  <action name="actionVTK_data">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>VTK data...</string>
   </property>
  </action>
  <action name="actionTo_Matlab">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>To Matlab...</string>
   </property>
  </action>
  <action name="actionTo_VTK">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>To VTK...</string>
   </property>
  </action>
  <action name="actionCALFEM_for_Python_documentation">
   <property name="text">
    <string>Documentation</string>
   </property>
  </action>
  <action name="actionVTK_tool_Guide">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>VTK tool Guide</string>
   </property>
  </action>
  <action name="actionShow_Origin">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Show Origin</string>
   </property>
  </action>
  <action name="actionShow_Elements">
   <property name="text">
    <string>Show Elements</string>
   </property>
  </action>
  <action name="actionShow_Elements_2">
   <property name="text">
    <string>Show Nodes</string>
   </property>
  </action>
  <action name="actionShow_Dofs">
   <property name="text">
    <string>Show Dofs</string>
   </property>
  </action>
  <action name="actionShow_Elements_3">
   <property name="text">
    <string>Show Elements</string>
   </property>
  </action>
  <action name="actionShow_Nodes">
   <property name="text">
    <string>Show Nodes</string>
   </property>
  </action>
  <action name="actionShow_Dofs_2">
   <property name="text">
    <string>Show Dofs</string>
   </property>
  </action>
  <action name="actionKeyboard_controls">
   <property name="text">
    <string>Keyboard controls...</string>
   </property>
  </action>
  <action name="actionStart_Animation">
   <property name="text">
    <string>Start Animation</string>
   </property>
  </action>
  <action name="actionEdit_Parameters">
   <property name="text">
    <string>Edit Parameters...</string>
   </property>
  </action>
  <action name="actionShow_browser">
   <property name="text">
    <string>Show browser</string>
   </property>
  </action>
  <action name="actionTrackball">
   <property name="text">
    <string>Trackball</string>
   </property>
  </action>
  <action name="actionSelection">
   <property name="text">
    <string>Selection</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
